src/components/DatasetSelect.tsx

```diff
@@ -83,13 +91,17 @@ export default function DatasetSelect({
   sortedDatasets.forEach((dataset) => {
     if (hasGroups) {
       const groupTitle = dataset.groupTitle || i18n.get("Others");
+      const groupDescription =
+        dataset.groupDescription || i18n.get("No Description provided");
       if (groupTitle !== lastGroupTitle) {
         items.push(
-          <Divider key={groupTitle}>
-            <Typography fontSize="small" color="text.secondary">
-              {groupTitle}
-            </Typography>
-          </Divider>,
+          <Tooltip arrow title={groupDescription}>
+            <Divider key={groupTitle}>
+              <Typography fontSize="small" color="text.secondary">
+                {groupTitle}
+              </Typography>
+            </Divider>
+          </Tooltip>,
```

Comment:
Make the tooltip wrapping conditional based on an available description, see above.